packages/react-components/react-breadcrumb/src/components/Breadcrumb/Breadcrumb.test.tsx

```diff
@@ -16,10 +16,13 @@ describe('Breadcrumb', () => {
     expect(result.container).toMatchInlineSnapshot(`
       <div>
         <nav
+          aria-label="breadcrumb"
```

Comment:
can this test use react testing library ? I'm not sure what the test actually test within this snapshot. we need to test if the focusing works not what is being rendered.

Reply:
This test was updated automatically. It's autogenerated one. I was going to add more tests for testing functionality